index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Week1</title>
+    <link rel="stylesheet" href="styles/style.css">
+</head>
+<body>
+    <header class = "header1">
+        <nav>
+            <ul>
+                <li><a href=""> <img src="images/swiggylogo.png" alt = "swiggylogo" class = "swiggylogo"> </a></li>
```

Comment:
Class names should be lowercased and hyphenated in case of multiple words

`swiggy-logo`